### ISSUE TITLE
Installation Wizard: Buttons are found but not clickable

### ISSUE THREAD

**Issue by GitHubTestUser5** [2gis/Winium.Desktop#115]
I am trying to automate UI of an installation wizard using Winium.Desktop tool.
And faced with a problem - the clicks on buttons work only on the first form of the installation wizard. For the next forms, button elements can be found, but clicks don't work. 

The installation wizard consists of a series of forms where main dialog is the same, but its content is dynamically changed after click on Next button. 

First of all the installer was started and I got an access to main dialog, found Next button and clicked on it. So far everything is good.  Then again found the main dialog (because it was updated), found Next button and tried to click on it, but click doesn't work. Note that AutomationId for Next button on those two first forms of an installation wizard is the same.

Is that an issue or limitation in Winium?

**Comment by NickAb** [comment]
Please try searching for all elements with the automation id and print out their number while on the screen with unclickable `Next` button. If there are more than one result for the id, then, most likely, when you are trying to find current Next button, you are still finding the Next button from previous step (screen). To deal with it, you will either have to specify the scope of Next button search more precisely by searching some ascendant of Next button that guaranteed to have uniq id and contain only current Next button. Or you can simple find all Next buttons and click current one by keeping track of step index and clicking the corresponding button from list by index.

For example, if you have tree somewhat like that:
```
Form
|_Step1
| |_Next
|_Step2
  |_Next
```
Then you should be searching for Next button starting from Step1 or Step2, not the Form. This will limit the scope and guarantee that you find the element you want. Alternatively you can find all Next buttons, save them to some list and then do list[1].click() to click the second one.

**Comment by GitHubTestUser5** [comment]
@NickAb, thanks for the answer. There is only one Next button on the screen with unclickable `Next `button, search for all elements with that Id returns only one found button element. Checked also isEnabled property of it and it is true. So the issue here is not in the same AutomationId of Next button on the installation wizard forms, because I have tried to click on "Change location" button the step 2 (where click on Next button doesn't work) and click on this button doesn't work also.

The issue on my side is related to screen resolution (it was set to 125% instead of 100%). After changing the screen resolution to 100%, the click on Next button on step1 and step2 works without problems.

So this is the same as #85
